### PR TITLE
Rdcc 418

### PR DESCRIPTION
**Pull request by ConorH994**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-418

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java [hunk 1]
```diff
@@ -86,8 +87,8 @@ public void setUp() {
         IdamOpenIdClient idamOpenIdClient = new IdamOpenIdClient(configProperties);
         IdamClient idamClient = new IdamClient(configProperties);
 
-        /*SerenityRest.proxy("proxyout.reform.hmcts.net", 8080);
-        RestAssured.proxy("proxyout.reform.hmcts.net", 8080);*/
+        SerenityRest.proxy("proxyout.reform.hmcts.net", 8080);
+        RestAssured.proxy("proxyout.reform.hmcts.net", 8080);
```
Comment (AdilOoze) [inline src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java]: Bro think this needs to be commented out?

File: src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ModifyRolesForUserTest.java [hunk 2]
```diff
@@ -95,7 +95,7 @@ public RequestSpecification generateBearerTokenForNonPuiManager() {
         }
     }
 
-    @Ignore// Ignoring until OpenId can be tested without hacking config file to point to p.r.
+    //@Ignore// Ignoring until OpenId can be tested without hacking config file to point to p.r.
```
Comment (AdilOoze) [inline src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ModifyRolesForUserTest.java]: I thought this was being ignored "until OpenId can be tested without hacking config file to point to p.r." - has this now been resolved?

File: src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java (hunk 1 above)
Comment (ConorH994) [reply src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java]: Do not merge until Func tests can be pointed at a User Profile PR with change status changes implemented

Comment (AdilOoze) [reply src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java]: 👍 grand

File: src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java [hunk 3]
```diff
@@ -133,7 +135,7 @@ public RequestSpecification generateBearerTokenForPuiManager() {
 
         bearerTokenForPuiUserManager = professionalApiClient.getMultipleAuthHeadersExternal(puiUserManager, firstName, lastName, userEmail);
 
-        // log.info("Bearer token generated for non pui user manager:::: " + bearerTokenForPuiUserManager);
+        log.info("Bearer token generated for non pui user manager:::: " + bearerTokenForPuiUserManager);
```
Comment (akiohmcts) [inline src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java]: We should not be logging the bearerToken as this is a Security Concern

**Pull request merged**